modules/launch_utils.py

```diff
@@ -68,7 +68,8 @@ def git_tag():
     try:
         return subprocess.check_output([git, "describe", "--tags"], shell=False, encoding='utf8').strip()
     except Exception:
-        return "<none>"
+        with open(os.path.dirname(os.path.abspath(__file__)) + "/../CHANGELOG.md") as file:
+            return next((line.strip() for line in file if line.strip()), "<none>")

```

Comment:
This would crash in an ugly way if the changelog is missing for some reason, or if the Python default encoding can't read the file (looking at you, Windows). Also, `os.path.join` or just `pathlib.Path`?

```suggestion
        try:
            changelog_md = pathlib.Path(__file__).parent.parent / "CHANGELOG.md"
            with changelog_md.open(encoding="utf-8") as file:
                return next((line.strip() for line in file if line.strip()), "<none>")
        except Exception:
            return "<none>"
```

Reply:
I suppose that's better
I did somehow assume that CHANGELOG.md will always be there but I guess it's better to not assume that
as for pathlib I was blind didn't realize we imported, though I only have os

Reply:
on I am right we didn't import that